src/local_scheduler/test/local_scheduler_tests.cc

```diff
@@ -210,12 +210,12 @@ TEST object_reconstruction_test(void) {
     int64_t task_assigned_size;
     local_scheduler_submit(worker, execution_spec);
     TaskSpec *task_assigned =
-        local_scheduler_get_task(worker, &task_assigned_size, true);
```

Comment:
Nice to have it removed, looking at this I had no idea what true means.  For literal arguments, especially booleans, I usually do
```
local_scheduler_get_task(worker, &task_assigned_size, /*param_name=*/true);
```